ppa/archive/admin.py

```diff
@@ -94,7 +94,7 @@ def save_related(self, request, form, formsets, change):
             request, form, formsets, change
         )
         digwork = DigitizedWork.objects.get(id=form.instance.pk)
-        digwork.index(params={"commitWithin": 10000})
+        digwork.index()
```

Comment:
@kmcelwee sorry, missed responding to this. Removing it because we dropped that argument when we adapted original solr code in PPA for parasolr. 

I don't follow your comment about rsync operating across the entire database, though.